FinniversKit/Sources/DNA/Color/UIColor+FinniversKit.swift

```diff
@@ -40,6 +40,10 @@ import UIKit
     public class var textTertiary: UIColor { Config.colorProvider.textTertiary }
     public class var textToast: UIColor { Config.colorProvider.textToast }
     public class var marketplaceNavigationBarIcon: UIColor { Config.colorProvider.marketplaceNavigationBarIcon }
+    public class var nmpBrandColorPrimary: UIColor { Config.colorProvider.nmpBrandColorPrimary }
```

Comment:
These new colors are used in the following PRs that will come. I have adopted the naming from Android